### DISCUSSION TITLE
Minimum JDK for openmdx 2.17.9

### DISCUSSION THREAD

**Discussion by nyemiks** [openmdx/openmdx#45] [Q&A] [answered]
I am currently on openmdx 2.15.7.  I am considering an upgrade to version 2.17.9. 

What is the minimum jdk for version 2.17.9 ?

 Can it run on java 8 ? 

Due to some other constraints I can't deploy on a java 11 environment.

**Comment by wfro64** [comment, marked as answer]
The binaries published on Maven are built with the target platform VERSION_1_8. See [here](https://github.com/openmdx/openmdx/blob/master/build.properties). So openMDX can run on Java 8.
